charts/emissary-ingress/templates/ambassador-agent.yaml

```diff
@@ -47,7 +47,7 @@ rules:
   verbs: [ "get", "list", "watch" ]
 - apiGroups: [""]
   resources: [ "secrets"]
-  verbs: [ "get", "create", "delete", "patch", "watch"]
+  verbs: [ "get", "create", "delete", "patch", "watch", "list"]
```

Comment:
just curious as to why the agent needs to list secrets and what secrets its listing?

Is there a new agent version that requires this?

Reply:
Yeah theres a new agent that requires this for 2 reasons, there is a change in the k8s library we are using that lists secrets instead of getting a specific one, and also now the agent secret can come from multiple sources, either emissary/ES install or telepresence, which is prefixed differently, so the agent lists the secrets and filters on the specific suffix of the secret since the exact naming isn't the same

Reply:
in that case you will also want to update the agent image version for the new version that requires this new action + update the chart changelog.